pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -49,10 +50,10 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		Short: "Creates a local Kubernetes cluster",
 		Long:  "Creates a local Kubernetes cluster using Docker container 'nodes'",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runE(logger, streams, flags)
+			return runE(cmd, logger, streams, flags)
 		},
 	}
-	cmd.Flags().StringVar(&flags.Name, "name", cluster.DefaultName, "cluster context name")
+	cmd.Flags().String("name", "", fmt.Sprintf(`the cluster name (default "%s")`, cluster.DefaultName))
```

Comment:
we should continue to document the default normally. it's possible to tell if a flag was set specifically.

Reply:
this comment still stands

Reply:
i added 
```
the cluster name. Overrides KIND_CLUSTER_NAME environment variable (default "kind")
```
in `usage` argument. is that not enough?